packages/bot/src/components/Icons.jsx

```diff
@@ -74,6 +74,14 @@ import trade_type_ticklow      from '../assets/icons/trade-types/ticklow.svg';
 import trade_type_unknown      from '../assets/icons/trade-types/unknown.svg';
 import trade_type_upordown     from '../assets/icons/trade-types/upordown.svg';
 
+// Transactions
+import contract_entry_spot      from '../assets/icons/ic-entryspot.svg';
+import contract_exit_spot       from '../assets/icons/ic-exitspot.svg';
+import contract_id              from '../assets/icons/ic-id.svg';
+import contract_pending         from '../assets/icons/contract-pending.svg';
+import contract_settled         from '../assets/icons/contract-settled.svg';
+import contract_buy_price       from '../assets/icons/buy-price.svg';
```

Comment:
I think `transaction-` prefix makes more sense considering these icons are used for transactions.

Reply:
@aaron-binary since its contract info its better this way, may we use it somewhere else. btw these all will be move to icons